src/pureimage.js

```diff
@@ -87,6 +87,28 @@ exports.encodeJPEGToStream = function(img, outstream) {
     });
 };
 
+/**
+ * Encode JPEG To Stream
+ * 
+ * Encode the JPEG image to output stream with q factor
+ *  
+ * @param {string} img       An object containing a raw buffer of the image data (`img.buffer`) along with the width(`img.width`) and height (`img.height`) of the image
+ * @param {Stream} outstream The stream to write the JPEG file to
+ * @param {number} q the wanted quality factor
```

Comment:
Might want to name this `quality` instead of `q`? Slightly move obvious what it means :)

Just a thought :)

Reply:
you are right. I changed the name of the parameter.

Reply:
It still appears to be `q` in the docblock on line 97. That's going to generate a lint error from esdoc if the docblock doesn't match the method signature.